Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -2,7 +2,7 @@
   "id": "a433f589-fce1-4460-9ee6-44a624aeb1fb",
   "name": "DownloadBuildArtifacts",
   "friendlyName": "Download Build Artifacts",
-  "description": "Download Build Artifacts",
+  "description": "Download artifacts from a file share e.g \\\\share\\drop",
```

Comment:
Why is the description for DownloadBuildArtifacts being changed?

Reply:
reverted.